src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -384,24 +384,7 @@ public string TransferEncoding
         }
 
 
-        public bool KeepAlive
-        {
-            get
-            {
-                return _webHeaderCollection[HttpKnownHeaderNames.KeepAlive] == bool.TrueString;
-            }
-            set
-            {
-                if (value)
-                {
-                    SetSpecialHeaders(HttpKnownHeaderNames.KeepAlive, bool.TrueString);
-                }
-                else
-                {
-                    SetSpecialHeaders(HttpKnownHeaderNames.KeepAlive, bool.FalseString);
-                }
-            }
-        } 
+        public bool KeepAlive { get; set; } = true;
```

Comment:
I don't understand this fix.

By the way, the KeepAlive property is a throwback to the early days of HTTP/1.0 where users needed to specify that they wanted to have the connection stay around.  Once HTTP/1.1 was done, the default semantics are for persistent connections and the connection keepalive request headers are by default sent. Thus, the defaults are usually for KeepAlive being true anyways.